src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -64,6 +65,11 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>Resource1.resx</DependentUpon>
     </Compile>
+    <Compile Include="Resources.Designer.cs">
```

Comment:
Just FYI, there's already a resource file in the vsix, resource1.resx. Your file name is better though. Also, resource1.resx was intended to be temporary. We needed to include an external file in the vsix, but I couldn't find the correct vsixmanifest settings. As a workaround, we embedded the file in resource1.resx until I figured out the manifest. That was a long way of saying, we should keep your resources.resx file and I'll remove resource1.resx in a future commit.